webapp/app/components/MessageList/MessageList.component.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import { Typography } from 'material-ui';
+
+import { Wrapper, Content, Loader } from './MessageList.styles';
+import { Message } from '../Message/Message.component';
+
+
+export class MessageList extends PureComponent {
+  static propTypes = {
+    messages: PropTypes.instanceOf(Map),
+  };
+
+  renderList = () => {
+    if (this.props.messages) {
+      if (this.props.messages.size) {
+        const sortedMessages = this.props.messages.sort();
```

Comment:
przerzucilbym to do renderMessages i w sumie tego sorta z map mozesz zapisac razem bez rozbiajania wtedy